package.json

```diff
@@ -21,6 +21,7 @@
     "lint": "eslint src/*.js test/*.js",
     "test-generate-mo": "msgfmt test/fixtures/latin13.po -o test/fixtures/latin13.mo & msgfmt test/fixtures/utf8.po -o test/fixtures/utf8.mo & msgfmt test/fixtures/obsolete.po -o test/fixtures/obsolete.mo",
     "test": "mocha",
+    "test:coverage": "npx c8 --check-coverage npm run test",
```

Comment:
Why not add `c8` as a dev dependency?

Reply:
because I would not want to add more dependencies, the ideal thing for this repo would be to switch to the [node's bundled test suite ](https://nodejs.org/api/test.html), which already has everything we need without installing nothing

Reply:
i.e. I don't think it is really limiting and since its use is limited to certain cases it can speed up the installation by not having it in the package.json. and then I am of the opinion that less is better

in any case if you think this will cause problems I will include it in the json package, alternatively I would proceed with the merge on the development branch so that we can know the coverage for each new PR